src/observer/management/commands/load_from_api.py

```diff
@@ -39,6 +42,21 @@ def handle(self, *args, **options):
                     # community is ~3 pages, digests is ~10
                     print("ignoring ID list, given content type doesn't support it.")
 
+            if days:
+                if targetlist != [LAX]:
+                    print("the '--days' parameter is only compatible with '--target=lax' right now")
+                    exit(1)
+
+                cutoff = utils.todt(datetime.now()) - timedelta(days=days)
+
+                def some_fn(result):
+                    "returns `True` when the versionDate for the given `result` falls within (now() - N days ago)."
+                    pubdate = utils.todt(result['versionDate'])
+                    return pubdate >= cutoff
+                #ingest_logic.download_regenerate_list(models.LAX_AJSON, some_fn)
+                ingest_logic.download_regenerate_article_list(some_fn)
+                exit(0)
```

Comment:
can the above be integrated in the normal operation so the daily update cron job doesn't have to be specified twice? if not, then specify twice in daily.sh